frontend/pages/frontend_sections/pb_efforts_section.py

```
@@ -74,124 +75,160 @@ def render_running_pb_section(data: StravaData, vars: Variables) -> None:
+                        .apply(lambda s: f"{s // 60}:{s % 60:02d}")
+                    )
+
+                # If plot metric is raw, create pace and time columns ready for plot
+                else:
+                    # Convert velocity (m/s) to pace in seconds per km
+                    all_effort_df['pace_sec'] = 1000 / all_effort_df['velocity_smooth']
+                    all_effort_df["distance"] = all_effort_df["distance"] / 1000
+
+                    # Calculate data point pace
+                    all_effort_df['pace'] = all_effort_df['velocity_smooth'].apply(
+                        lambda x: f"{int((1000/x)//60)}:{int(round((1000/x) % 60)):02d}" if x > 0 else "Invalid speed"
```

Ruff (0.15.6)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)